inst/extdata/example_release.yaml

```diff
@@ -0,0 +1,62 @@
+
+meddle-version: "0.0.3"
+document-created: "20161022"
+document-creator: "Jordan S Read"
+
+# ----data release----
+authors: ["Alison P Appling","Jordan S Read","Luke A Winslow", "Arroita", "Emily Bernhardt", "Natalie Griffiths", "Robert Hall", "Judson Harvey", "Heffernan", "Emily Stanley", "Edward Stets", "Charles Yackulic"]
+pubdate: 2017
+title: >-
+  Data release: A national database for river metabolism (2007-present)
+doi: http://dx.doi.org/10.5066/REPLACE.w.REAL
```

Comment:
a handy utility would be if `render` threw warnings for each element that used some standard note-to-self syntax (e.g., "REPLACE.w.REAL") so that you'd know when you were done and what was left to fill in.